posthog/tasks/scheduled.py

```diff
@@ -246,6 +247,12 @@ def setup_periodic_tasks(sender: Celery, **kwargs: Any) -> None:
         name="update survey iteration based on date",
     )
 
+    sender.add_periodic_task(
+        crontab(hour="*", minute="20"),
```

Comment:
Should we start with each hour to begin with?

Reply:
good call, tbh it should be configurable, someone self-hosting might want to set this to every second for example 🙈

Reply:
Huh? Do you mean running the task at the beginning of each hour, not at 20 minutes?

Reply:
Sorry, read that line too quickly. We can leave it as is.